### DISCUSSION TITLE
On a Mac, how to keep Uno up to date ?

### DISCUSSION THREAD

**Discussion by stephenhauck** [unoplatform/uno#15440] [Q&A] [answered]
Is there a script I should be running weekly or something to keep the Uno platform up to date ?

**Comment by jeromelaban** [comment]
Thanks for the question. If you're using VS Code, at this time, C# Dev Kit does not yet provide a way to update nuget packages from the IDE.

Instead, you can use [`dotnet-outdated`](https://github.com/dotnet-outdated/dotnet-outdated) which will allow you to upgrade packages from your solution.

**Reply by jeromelaban** [reply]
_(let's keep conversations in the same thread)_

> That will work for the app .. I'm talking about the Uno SDK itself ...

Just to be sure, you're talking about the version specified in the `global.json` file, correct?

**Reply by stephenhauck** [reply]
No ... I'm talking about Uno overall ... do I just keep running uno check or something ?

**Reply by jeromelaban** [reply, marked as answer]
I see, thanks.

Updating Uno Platform is mostly about the nuget packages and the `Uno.SDK` line in the `global.json`. Using Uno.check will only validate that you're setup with external dependencies or tools to their most recent/supported versions (like .NET, Android, VS 2022, Xcode, etc...). Extensions for VS and VS Code automatically update.

**Comment by stephenhauck** [comment]
Thanks for the response .. 
That will work for the app .. I'm talking about the Uno SDK itself ...
